### PR TITLE
Add settings and middleware for demo

### PR DESCRIPTION
**Pull request by oscarmcm**
Final step to complete #1263 - related to https://github.com/django-wiki/django-wiki/issues/1044#issuecomment-612394655

### CODE REVIEW

File: testproject/testproject/settings/demo.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from .base import *  # noqa
+
+DEBUG = False
+
+INTERNAL_IPS = []
```
Comment (oscarmcm) [inline testproject/testproject/settings/demo.py:5]: @benjaoming Not sure if here we'll leave a fixed value or read it from the environment var.

File: testproject/testproject/fixtures/fixtures.json [hunk 2]
```diff
@@ -155,8 +155,8 @@
       "model": "sites.site",
       "pk": 1,
       "fields": {
-          "domain": "example.com",
-          "name": "example.com"
+          "domain": "django-wiki.org",
```
Comment (benjaoming) [inline testproject/testproject/fixtures/fixtures.json:158]:
```suggestion
          "domain": "demo.django-wiki.org",
```

File: testproject/testproject/settings/demo.py (hunk 1 above)
Comment (benjaoming) [reply testproject/testproject/settings/demo.py:5]: These parts are injected by the deployment script so they can be ignored. Unless you know a good default that's needed for local deployments of the demo :+1:

**Review by benjaoming — APPROVED** [review]

**Comment by oscarmcm** [comment]
@benjaoming I believe this is ready to go!

File: testproject/testproject/fixtures/fixtures.json [hunk 3]
```diff
@@ -119,7 +119,7 @@
       "model": "auth.user",
       "pk": 1,
       "fields": {
-          "password": "pbkdf2_sha256$390000$IICRDKwC5zBSkzbCfpkUpx$xxVdvVYU2BkEXRbmoyDBcy2paLXAGrkt4COJslZBOgU=",
+          "password": "",
```
Comment (benjaoming) [inline testproject/testproject/fixtures/fixtures.json:122]: Does this make the password "unusable"?

Comment (oscarmcm) [reply testproject/testproject/fixtures/fixtures.json:122]: yups!

**Comment by oscarmcm** [comment]
@benjaoming I've just added support in the fixtures for the image plugin, so there's no real need now to use a custom management command.

image

Related to the "locked" word, seems to be related to this:
https://github.com/django-wiki/django-wiki/blob/main/src/wiki/templates/wiki/article.html#L24

More than that, this should be ready to merge, lmk.

**Comment by benjaoming** [comment]
Deployed and working :+1: https://demo.django-wiki.org/

**Comment by benjaoming** [comment]
Something up with the admin user, though.. it doesn't seem to have any admin rights:

image

**Comment by oscarmcm** [comment]
I just left the Admin user to be able to add/edit new pages. Not sure what else we must be covering.

**Comment by benjaoming** [comment]
I couldn't get that to work, `admin` doesn't seem to own `use-this-space` so cannot add new pages nor edit anything existing stuff.

**Comment by oscarmcm** [comment]
ohhh I see, there's an small issue in the fixtures

https://github.com/django-wiki/django-wiki/blob/main/testproject/testproject/fixtures/fixtures.json#L297

The article is owned by `djangowiki` super user, changing that to `2` should fix it.

**Pull request merged**